home-music/home-music.html

```diff
@@ -1,13 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="home-music.css" />
-  <title>GAMSIC</title>
-</head>
-<body>
-  <script src="home-music.js"></script>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Alf font from google fonts library -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Alef:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- Remix icons library -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css"
+      rel="stylesheet"
+    />
+    <!-- Icon link -->
+    <link
+      rel="icon"
+      type="image/svg"
+      href="../assets/images/game-controller.svg"
+    />
+    <link rel="stylesheet" href="home-music.css" />
+    <title>GAMSIC</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="head icons">
+        <a href="../index.html" class="landing-page"
+          ><i class="ri-arrow-left-s-line arrow"></i
+        ></a>
+        <i class="ri-sun-fill sun"></i>
+      </header>
+      <main class="content">
+        <img
+          src="../assets/images/image 3.png"
+          alt="music roll"
+          id="music-pic"
+          width="569px"
+        />
+        <div class="rest-cont">
+          <div class="headings">
+            <h1 class="main-head">GAMSIC</h1>
+            <h4 class="second-head">MUSIC</h4>
+          </div>
+          <p class="paragraph">
+            A web page that displays product details about the game console.
+          </p>
+          <button class="learn-more">
+            <a href="../music/music.html" class="music-page">Learn More!</a>
+          </button>
```

Comment:
the anchor tag should wrap the button tag so that the user can press anywhere in the button and move to the next page, not just the text

Reply:
I thought about it, Imma fix it.